### ISSUE TITLE
As of 1.3.3, I need to build three times

### ISSUE THREAD

**Issue by kvz** [rhash/RHash#32]
This is how I have to build now when I want to install it to `/srv/current/stack`:

```bash
install_dir=/srv/current/stack
make -j$(getconf _NPROCESSORS_ONLN) clean PREFIX= DESTDIR=${install_dir}

# For some reason, building three times works.. :'(
make -j$(getconf _NPROCESSORS_ONLN) install PREFIX= DESTDIR=${install_dir} \
  || make -j$(getconf _NPROCESSORS_ONLN) install PREFIX= DESTDIR=${install_dir} \
  || make -j$(getconf _NPROCESSORS_ONLN) install PREFIX= DESTDIR=${install_dir}
```

The first time it fails on:

```
gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations rhash.c -o rhash.o
install: cannot stat ‘rhash’: No such file or directory
make[1]: *** [install-binary] Error 1
make[1]: *** Waiting for unfinished jobs....
```

Then the second time it fails on:

```
install -d /srv/current/stack/etc
install -m 755 rhash /srv/current/stack/bin
install -m 644 dist/rhash.1 /srv/current/stack/share/man/man1/rhash.1
sed -e 's/\x0D//g' dist/rhashrc.sample > rc.tmp && install -m 644 rc.tmp /srv/current/stack/etc/rhashrc
install: cannot stat ‘rhash’: No such file or directory
make[1]: *** [install-binary] Error 1
make[1]: *** Waiting for unfinished jobs....
```

And the third time, it succeeds.

`1.3.2` did not have this problem

**Comment by rhash** [comment]
Thanks for reporting it.

Fixed by commit 01525eec84343d7e3cec5145885b9f26df8ace46.

**Comment by rhash** [comment]
Fixed in v1.3.5.
